packages/dds/tree/docs/storage/treeStorage.md

```diff
@@ -0,0 +1,801 @@
+# Fluid DDS Storage: Virtualization and Incrementality
```

Comment:
I'm already pretty happy with this (we did many offline revisions), but I did notice that sometimes the graphics failed to load for me and other viewers. Any idea on why?

Reply:
How are you viewing the file?

Reply:
Via GitHub, which I assume is the primary medium.

Reply:
Sorry for ninja edit, see above

Reply:
I do see them, typically roughly 80% of them load, and I get an error box for the rest. If I refresh, it fixes it. I know several other people mentioned the same thing. It's probably fine.